pkg/util/blockdevice/blockdevice_linux.go

```diff
@@ -85,6 +85,16 @@ func checkBlockDeviceSize(devicePath string, deviceMountPath string, newSize int
 	return nil
 }
 
+func triggerRescan(blockDeviceRescanPath string) error {
+	klog.V(4).Infof("Rescanning %q block device geometry", blockDeviceRescanPath)
+	err := os.WriteFile(blockDeviceRescanPath, []byte{'1'}, 0666)
```

Comment:
I haven't try this before, but 0666 is convention for such operation?

Reply:
It is a good question and I am not sure to be honest if all these permissions are required, I just reused the logic that already was here. However I saw that the original author of this rescan logic was @kayrus so maybe he could make us wiser

Reply:
I took this from upstream k8s libs for similar operations.